Oracle/condition.sql

```diff
@@ -54,6 +54,38 @@ CREATE TABLE SOURCEFACT2  (
 	C_FULLNAME   	VARCHAR2(700) NOT NULL
 	)
 /
+
+drop table dxsource_fact
```

Comment:
what if this table doesn't exist?

I think there's a delete_table stored procedure (not sure of the name) that is the norm in the i2p-transform code. (We should probably use it elsewhere too.)